theme/assets/js/templates/store/item.js

```diff
@@ -6,6 +6,7 @@ document.addEventListener('alpine:init', () => {
         product: {},
         locations: [],
         formData: {},
+        selectedVariationPrice: null,
```

Comment:
product store now indicates the Selected Variation Price, which updates when new choices are made in the Product Form

Reply:
we could include all the data about the Selected Variation (ID etc) but it seemed unnecessary for now